open-source-terms.md

```diff
@@ -0,0 +1,259 @@
+    any agreement with a third party.
+
+3.  You will not disclose information that you do not have the right to
+    disclose, such as confidential information of others.
+
+4.  You will not copy or share any personally identifiable information of
+    any other person without their specific permission.
+
+5.  You will not violate any applicable law.
+
+6.  You will not use or attempt to use another person's Account without
+    their specific permission.
+
+7.  You will not send advertisements, chain letters, or other
+    solicitations via npm Services.
+
+8.  You will not automate access to, use, or monitor the Website, such
+    as with a web crawler, browser plug-in or add-on, or other computer
+    program that is not a web browser. You may replicate data from the
+    Public Registry using the Public APIs per this Agreement.
+
+9.  You will not use npm Services to send e-mail to distribution lists,
+    newsgroups, or group mail aliases.
+
+10. You will not submit material containing malicious computer code,
+    such as computer viruses, computer worms, rootkits, back doors,
+    adware, or spyware, to npm Services in such a way that you should
+    expect other users of npm Services to unwittingly execute that
+    malicious code.
+
+11. You will not falsely imply that you are affiliated with or endorsed
+    by npm.
+
+12. You will not operate illegal schemes, such as pyramid schemes, via
+    npm Services.
+
+13. You will not deep-hyperlink to images or other non-hypertext content
+    served by npm Services.
+
+14. You will not remove any marking indicating proprietary ownership
+    from any material got via npm Services.
+
+15. You will not display any portion of the Website via an HTML IFRAME.
+
+16. You will not disable, avoid, or circumvent any security or access
+    restrictions of npm Services, or access parts of npm Services not
+    intended for access by you.
+
+17. You will not strain infrastructure of npm Services with an
+    unreasonable volume of requests, or requests designed to impose an
+    unreasonable load on IT systems underlying npm Services.
+
+18. You will not encourage or assist any other person in violation of
+    "Acceptable Use".
+
+## Enforcement of Acceptable Use
+
+npm may investigate and prosecute violations of this Agreement to the fullest legal extent. npm may notify and cooperate with law enforcement authorities in prosecuting violations of this Agreement.
+
+## Your Account
+
+You must create and log into an Account to access features of some npm Services, including npm Open Source.
+
+To create an Account, you must provide certain information about yourself, as required by the account creation form on the Website. If you create an Account, you will provide, at a minimum, a valid e-mail address. You will keep that e-mail address up-to-date. You will not impersonate any other individual. You may delete your Account at any time using the Website.
```

Comment:
We don't currently have a "delete my account" feature on the website, but we can do this via emailing support.